.github/workflows/build-test-publish.yml

```diff
@@ -14,44 +14,47 @@ jobs:
   test:
     name: Test with Python ${{matrix.python_version}} on ${{matrix.os}}
     runs-on: ${{matrix.os}}
+    env:
+      SKIP_LONG_RUNNING_TESTS: ${{ github.event_name == 'pull_request' && github.event.pull_request.draft == true }}
     strategy:
       fail-fast: false
       matrix:
-        os: [ubuntu-latest]
+        os: [ubuntu-latest, windows-latest]
         python_version:
           - "3.11"
           - "3.12"
           - "3.13"
-        include:
-          - os: windows-latest
-            python_version: "3.11"
```

Comment:
I don't think we need to run Windows tests for all Python versions, the minimum supported one should be enough. The other Python versions are already covered by the Linux tests.

Reasons being that a) Windows tests usually run a little slower compared to Linux, and b) they consume twice the CI time on Github (though this is a public repo, so we don't actually have to pay that).